front/src/routes/scene/edit-scene/actions/SendMessageCameraParams.jsx

```diff
@@ -0,0 +1,151 @@
+import Select from 'react-select';
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import { Text } from 'preact-i18n';
+
+import TextWithVariablesInjected from '../../../../components/scene/TextWithVariablesInjected';
+
+const helpTextStyle = {
```

Comment:
We try to avoid this now in favor of bootstrap native styles (mb-1 for example), or style.css files

https://community.gladysassistant.com/t/frontend-amelioration-diverses-experience-developpement/8125

Reply:
Fixed